src/ServiceManager.js

```diff
@@ -0,0 +1,45 @@
+const boxen = require('boxen')
```

Comment:
What does this look like when it works?

Reply:
<img width="742" alt="screen shot 2018-04-26 at 2 14 38 am" src="https://user-images.githubusercontent.com/32407/39288845-ae603c62-48f7-11e8-801f-ee2464ecc030.png">

The cool thing about this is we can have client-js start browsers for the same services, and get hostnames that make sense from your local laptop - which might show talon.local, or crosby.cluster..